cartoframes/io/carto.py

```diff
@@ -227,14 +232,16 @@ def copy_table(table_name, new_table_name, credentials=None, if_exists='fail', l
         new_table_name(str, optional): name for the new table.
         credentials (:py:class:`Credentials <cartoframes.auth.Credentials>`, optional):
             instance of Credentials (username, api_key, etc).
-        if_exists (str, optional): 'fail', 'replace'. Default is 'fail'.
+        if_exists (str, optional): 'fail', 'replace', 'append'. Default is 'fail'.
     """
     if not isinstance(table_name, str):
-        raise ValueError('Wrong table name. You should provide a valid string.')
+        raise ValueError('Wrong table name. You should provide a valid table name.')
```

Comment:
Could we add a `:raises ValueError: When the table name provided is wrong or the `if_exists` param is not valid` to the apidocs?